server/src/config.js

```diff
@@ -281,6 +274,13 @@ var conf = convict({
     default: "",
     env: "CDN",
     arg: "cdn"
+  },
+  userSettings: {
```

Comment:
Let's name this something with the name "enable" in it, like "enableUserSettings" – then it's clearer that it's a feature flag.